client/components/SampleCensus.js

```diff
@@ -2,330 +2,159 @@ import React from 'react';
-                        defaultMessage="Children, related or unrelated, such as newborn babies, grandchildren, or foster children"
-                      />
-                    </label>
-                  </li>
-                  <li>
-                    <label>
-                      <SampleCensusCheckbox
-                      />
-                      <FormattedMessage
-                        id="components.SampleCensus.form.question2.fields.related-adults"
-                        defaultMessage="Relatives, such as adult children, cousins, or in-laws"
-                      />
-                    </label>
-                  </li>
-                  <li>
-                    <label>
-                      <SampleCensusCheckbox
-                      />
-                      <FormattedMessage
-                        id="components.SampleCensus.form.question2.fields.nonrelatives"
-                        defaultMessage="Nonrelatives, such as roommates or live-in babysitters"
-                      />
-                    </label>
-                  </li>
-                  <li>
-                    <label>
-                      <SampleCensusCheckbox
-                      />
-                      <FormattedMessage
-                        id="components.SampleCensus.form.question2.fields.staying-temporarily"
-                        defaultMessage="People staying here temporarily"
-                      />
-                    </label>
-                  </li>
-                  <li>
-                    <label>
-                      <SampleCensusCheckbox
-                      />
-                      <FormattedMessage
-                        id="components.SampleCensus.form.question2.fields.no-additional"
-                        defaultMessage="No additional people"
-                      />
-                    </label>
-                  </li>
-                </ul>
-              </li>
-              <li>
-                <header>
-                  <h3 className="c_sample-census__form__question-header">
-                    <FormattedMessage
-                      id="components.SampleCensus.form.question3.header"
-                      defaultMessage="Is this house, apartment, or mobile home"
-                    />
-                  </h3>
-                  &mdash;
-                  <span className="c_sample-census__form__question-instructions">
-                    <FormattedMessage
-                      id="components.SampleCensus.form.markOne"
-                      defaultMessage="Mark {markBoxImage} ONE box"
-                      values={{
-                        markBoxImage: (
-                          <SampleCensusCheckbox
-                            checked
-                            readOnly
-                          />
-                        )
-                      }}
-                    />
-                  </span>
-                </header>
+    //REMOVE ME
```

Comment:
? still need to remove it?

Reply:
At the moment this is creating the list of 9 placeholder questions to be rendered, so we still need it if we want to have a demonstrable page on Heroku. I am trying to figure out a more concise way to dynamically create FormattedMessage components within the CensusQuestionCard while still getting their default text translated.

Reply:
+1 k will merge for now